packages/next/src/server/next-server.ts

```diff
@@ -1512,36 +1551,63 @@ export default class NextNodeServer extends BaseServer<
       middleware: true,
     })
 
+    const method = (params.request.method || 'GET').toUpperCase()
+    const requestData = {
+      headers: params.request.headers,
+      method,
+      nextConfig: {
+        basePath: this.nextConfig.basePath,
+        i18n: this.nextConfig.i18n,
+        trailingSlash: this.nextConfig.trailingSlash,
+        experimental: this.nextConfig.experimental,
+      },
+      url: url,
+      page,
+      body:
+        method !== 'GET' && method !== 'HEAD'
+          ? (getRequestMeta(params.request, 'clonableBody') as any)
+          : undefined,
```

Comment:
Is this a behavior change? IIRC while GET semantically does not allow a body, it is not forbidden by spec.

Reply:
Node Request instance throws error if you try passing body with GET or HEAD requests.